docs/components/authentication/sign-up.mdx

```diff
@@ -85,12 +85,48 @@ Below is basic implementation of the `<SignUp />` component. You can use this as
   </Tab>
 
   <Tab>
-    ```js filename="sign-up.js"
-    window.Clerk.mountSignUp(
-      document.getElementById("sign-up")
-    );
-    window.Clerk.openSignUp();
-    ```
+    In the example below, the [`mountSignUp()`](/docs/references/javascript/clerk/sign-up#mount-sign-up) method is used to render the `<SignUp />` component to a `<div>` element with `id="sign-up"`.
```

Comment:
We should avoid words like "below" because non-sighted people don't necessarily perceive this information as being above or below. I prefer "The following example".

I'd also suggest avoiding describing the code in detail like this. They can read the code to see the implementation details, and what if the specifics of the example change later but we forget to update this text? Let's just tell them what the example does: "The following example renders the `<SignUp />` component using `mountSignUp()`."